src/printer.js

```diff
@@ -758,10 +762,10 @@ function genericPrintNoParens(path, options, print) {
 
       var isParentForLoop = namedTypes.ForStatement.check(parentNode) ||
         namedTypes.ForInStatement.check(parentNode) |
```

Comment:
What's neat is that I noticed it especially because prettier changed the indentation :) At least I think it's because it has a different precedence level so if was formatted differently.